### PR TITLE
Fix alignment of currency on Box Office

### PR DESCRIPTION
**Pull request by KevinDurand974**
From: https://github.com/blockframes/blockframes/issues/7978

* * *

Before
preview before

After
preview after

### CODE REVIEW

**Comment by KevinDurand974** [comment]
Check if we can remove the mat-icon by a simple text with a condition on '$' or '€'

**Comment by KevinDurand974** [comment]
Now with code optimization :

preview

File: libs/movie/src/lib/movie/form/additional-information/additional-information.component.html [hunk 1]
```diff
@@ -58,9 +58,10 @@ <h3>Box Office</h3>
           <ng-template itemRef let-boxOffice>
             <span>
               {{ boxOffice.territory | toLabel: 'territories' }} -
-              <span *ngIf="boxOffice.unit === 'eur'">€ {{ boxOffice.value }}</span>
-              <span *ngIf="boxOffice.unit === 'usd'">$ {{ boxOffice.value }}</span>
-              <span *ngIf="boxOffice.unit === 'admissions'">{{ boxOffice.value }} Admissions</span>
+              <ng-container *ngIf="boxOffice.unit !== 'admissions' else admissions">
+                {{ boxOffice.unit === 'eur' ? '€' : '$'}} {{ boxOffice.value }}
+              </ng-container>
+              <ng-template #admissions>{{ boxOffice.value }} Admissions</ng-template>
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/additional-information/additional-information.component.html:64]: you have quite the same code in `libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html`
```
<missing-control [control]="office.get('unit')" [link]="link">
                <ng-template let-unit>
                  <ng-container *ngIf="unit === ('eur' || 'usd'); else admissions">
                    <mat-icon [svgIcon]="unit | uppercase" id="currency-icon"></mat-icon>
                  </ng-container>
                  <ng-template #admissions>
                    <span>Admissions</span>
                  </ng-template>
                </ng-template>
              </missing-control>
```

Comment (phpgeekfr) [reply libs/movie/src/lib/movie/form/additional-information/additional-information.component.html:64]: and remove unused css 
```
.currency-icon {
  margin-top: 8px;
  height: 16px;
}
```

Comment (phpgeekfr) [reply libs/movie/src/lib/movie/form/additional-information/additional-information.component.html:64]: same for libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.scss

Comment (phpgeekfr) [reply libs/movie/src/lib/movie/form/additional-information/additional-information.component.html:64]: also, if I choose $ in box office, in summary I have :
image

**Review by phpgeekfr — APPROVED** [review]

**Pull request merged**